src/app/helpers/latexService.js

```diff
@@ -45,6 +46,11 @@ function svg2png(svgString) {
     });
 }
 
+function png2base64(pngString) {
```

Comment:
@vrayulu We have added this method to convert the formula to base64. 
cc: @vinukumar-vs